src/fontra/views/editor/panel-selection-info.js

```diff
@@ -463,6 +482,7 @@ export default class SelectionInfoPanel extends Panel {
     }
 
     this.infoForm.onFieldChange = async (fieldItem, value, valueStream) => {
+      fieldItem.initialValue = value;
```

Comment:
@justvanrossum  This is the line of code, where I store the latest value.

Reply:
That is not correct: we shouldn't use the field item that way.